include/envoy/stats/stats.h

```diff
@@ -194,6 +194,11 @@ class Sink {
    */
   virtual void flushGauge(const Gauge& gauge, uint64_t value) PURE;
 
+  /**
+   * Flush a histogram.
+   */
+  virtual void flushHistogram(const Histogram&){};
```

Comment:
PURE (have every sink implement, even if they ignore).

Reply:
yes.. will change them. added them like that so that i can quickly test with out changing tests.